### DISCUSSION TITLE
The filter won't show in the filter list.

### DISCUSSION THREAD

**Discussion by santosh** [royshil/obs-backgroundremoval#507] [General]
I am using Ubuntu, and am downloading .deb file from the release page. I am installing and opening up the obs studio.

When I try to add filter on my Video Capture device (v4l2), I only see **Video Delay (Async)**.

I don't understand why. Have any of you encountered this before?

**Comment by umireon** [comment]
@santosh Can you open an issue for further information?
https://github.com/occ-ai/obs-backgroundremoval/issues/new/choose

**Reply by santosh** [reply]
Why does building on linux requires zsh? This plugin is not even a CLI tool.

With most of the distros having bash as default shell, why does build script require zsh?

**Reply by umireon** [reply]
Please open an issue as I requested.
